lib/capybara/node/element.rb

```diff
@@ -43,14 +43,15 @@ def native
 
       ##
       #
-      # Retrieve the text of the element. If `Capybara.ignore_hidden_elements`
+      # Retrieve the text of the element. If {Capybara.configure ignore_hidden_elements}
       # is `true`, which it is by default, then this will return only text
       # which is visible. The exact semantics of this may differ between
       # drivers, but generally any text within elements with `display:none` is
       # ignored. This behaviour can be overridden by passing `:all` to this
       # method.
       #
       # @param type [:all, :visible]  Whether to return only visible or all text
+      # @param normalize_ws [Boolean] When `true`, remove all whitespace from the returning text
```

Comment:
[note] I think this `normalize_ws` is required and add it, but I'm worried whether this modification is correct… 🤔

Reply:
`normalize_ws` is something that was added as a stopgap to add migration from Capybara 2 to 3.  It wasn't added to the documentation because its use really shouldn't be encouraged and will probably go away in Capybara 4.  Also it doesn't remove all whitespace, it normalizes the whitespace (collapses multiple whitespace to one, strips beginning and end, etc)

Reply:
Oh, I see. 👍 
But `normailze_ws` is yet confused because the method signature contains it.

![image](https://user-images.githubusercontent.com/473530/58309237-a7cfdb00-7e3e-11e9-9221-ce8c640feff0.png)
https://www.rubydoc.info/gems/capybara/3.20.2/Capybara/Node/Element#text-instance_method

How about removing it from doc explicitly by using `@overload`? Like this: ⬇️ 

```ruby
# @overload text(type = nil)
#   @param type [:all, :visible]  Whether to return only visible or all text
#   @return [String]              The text of the element
def text(type = nil, normalize_ws: false)
```

I try it on my local: ⬇️ 
![image](https://user-images.githubusercontent.com/473530/58309414-1e6cd880-7e3f-11e9-8275-7b25211e232e.png)

Reply:
I think for now I just want to leave this the way it was, shows in the method but isn't otherwise documented.